Lib/glyphsLib/builder/transformations/propagate_anchors.py

```diff
@@ -211,6 +211,9 @@ def anchors_traversing_components(
         component_transform = Transform(*component.transform)
         xscale, yscale = get_xy_rotation(component_transform)
         for anchor in anchors:
+            # skip contextual anchors
+            if anchor.name.startswith("*") and "GPOS_Context" in anchor.userData:
```

Comment:
I’m being a bit conservative here and testing specifically for contextual anchors (`GPOS_Context`), but it might be that GlyphsApp does not propagate any special anchors, but I have no way of checking that.

Reply:
Glyphs traverses all anchors. You can check in Glyphs by running this in the Macro panel:
```
print(Layer.anchorsTraversingComponents())
```

Reply:
I was testing by checking the generated feature file (using FEA Export plugin), and it had contextual anchor lookups only for glyphs that explicitly had them. `Layer.anchorsTraversingComponents()` does indeed propagate contextual anchors. Now I checked the GlyphsApp-generated font and it have the anchors propagated as well, so I guess it is the plug that mislead me.